pkg/model/local_target.go

```diff
@@ -0,0 +1,66 @@
+package model
+
+import (
+	"fmt"
+
+	"github.com/windmilleng/tilt/internal/sliceutils"
+)
+
+type LocalTarget struct {
+	Cmd  Cmd
+	Deps []string
+
+	dockerignores []Dockerignore
+	repos         []LocalGitRepo
+}
+
+var _ TargetSpec = LocalTarget{}
+
+func NewLocalTarget(cmd Cmd, deps []string) LocalTarget {
+	return LocalTarget{Cmd: cmd, Deps: deps}
+}
+
+func (lt LocalTarget) WithRepos(repos []LocalGitRepo) LocalTarget {
+	lt.repos = append(append([]LocalGitRepo{}, lt.repos...), repos...)
+	return lt
+}
+
+func (lt LocalTarget) WithDockerignores(dockerignores []Dockerignore) LocalTarget {
```

Comment:
Are Dockerignores needed? Or GitRepos?

Reply:
yeah, because we set up watches for the deps just like we do for custom_build.deps